internal/buffer/buffer.go

```diff
@@ -64,6 +64,8 @@ var (
 	// BTStdout is a buffer that only writes to stdout
 	// when closed
 	BTStdout = BufType{6, false, true, true}
+	// BTDefaultNoSyntax is a default buffer with disabled syntax highlighting
+	BTDefaultNoSyntax = BufType{0, false, false, false}
```

Comment:
I could have used `BTScratch` too, but this would change the current approach of not storing this kind of buffer, so I decided to create a new one to explicit highlight what it is used for.
Currently I've no feeling, if defining his `Kind` to be of the same type as `BTDefault` was a wise choice...

Reply:
> I think the whole approach is wrong. Why not decouple the backup functionality from the `Buffer` (and move it outside the `buffer` package) instead?

That's also a possibility. Currently I've no overview if it's worth the effort. At least we wouldn't need the `Buffer` as intermediate step and thus no new type.

> On second thought: actually maybe it's ok (maybe even useful) that they use same backup file, as long as we make sure that they don't conflict? (Do we?)

At least I didn't focus on that yet. So most probably we don't.

> [...], looks like we also have at least the following issues with backups: [...]

I will add it to the TODOs.

Reply:
> Currently I've no overview if it's worth the effort.

I think it is. Assuming that we want to implement this feature (generalized backups) at all, let's implement it cleanly, not via ugly design shortcuts.

...Though the higher-level question worth thinking about: is implementing this feature (i.e. "safe save" not just for buffers but also for internal files) worth the effort in the first place? Seems like it is, occasionally wiped `settings.json` is very annoying.

Partially this is a consequence of the (IMHO unfortunate) historical decision to let micro write its config files automatically (e.g. vim and nano don't write their vimrc and nanorc, so they have no dilemma how to write them safely)... OTOH even regardless of `settings.json` and `buffers.json` there are also those serialized files in `~/.config/micro/buffers/` which would need to be written by micro anyway...